fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/MalformedRdfExceptionMapper.java

```diff
@@ -39,6 +39,10 @@
     public Response toResponse(final MalformedRdfException e) {
         final Link link = Link.fromUri(getConstraintUri(e)).rel(CONSTRAINED_BY.getURI()).build();
         final String msg = e.getMessage();
+
+        if (msg.indexOf("given RDF is out-of-date") != -1) {
```

Comment:
This ExceptionMapper does not apply to the scenario described in the ticket. Instead, `ServerManagedPropertyExceptionMapper` is used... but does not appear to require any updates.